### ISSUE TITLE
Script upgrade fails to execute needed Roundcube scripts

### ISSUE THREAD

**Issue by PitWenkin** [virtualmin/virtualmin-gpl#740]
When upgrading RoundCube via [edit_newscripts.cgi](https://github.com/virtualmin/virtualmin-gpl/blob/b50a6ffe6fc7d8c9e60f716ed16d2cfac2e89bd8/edit_newscripts.cgi) some database changes are not and/or were not applied.

RoundCube's [update.sh](https://github.com/roundcube/roundcubemail/blob/57613362533b0abf1278399fcc57c1fd4fc1239d/bin/update.sh) needs do be executed for database schemas to be updated.

As far as I remember, there was no warning to check for such a script in the virtualmin interface.

Executing the script manually returns the following:
```
root@server:/home/example/public_html/folder# ./bin/update.sh 
What version are you upgrading from? Type '?' if you don't know.
?
WARNING: Replaced config options:
(These config options have been replaced or renamed)
- 'default_host' was replaced by 'imap_host'
- 'smtp_server' was replaced by 'smtp_host'
Do you want me to fix your local configuration? (y/N)
n
Please update your config files manually according to the above messages.
Executing database schema update.
Updating database schema (2021081000)... [OK]
Updating database schema (2021100300)... [OK]
Updating database schema (2022081200)... [OK]
This instance of Roundcube is up-to-date.
Have fun!
```

On this instance there seems to be database upgrades missing since October of 2021?

I'm using the `Upgrade Notification`-feature to find about new RoundCube version, by being notified from virtualmin itself. Therefor the last few upgrades to RoundCube have only been applied once virtualmin notified my about them, and they have only been done by using the functions provided by virtualmin.

(This Issue is somewhat related to #739 )

**Comment by iliajie** [comment]
Hello,

Thanks for the heads up! This was fixed in this https://github.com/virtualmin/virtualmin-gpl/commit/ed8485c99625a4c36f0a168c0cd8f180734913de patch.

**Comment by PitWenkin** [comment]
I got a notification today about [RoundCube 1.6.6](https://github.com/roundcube/roundcubemail/releases/tag/1.6.6)  being available.

As far as I see there shouldn't be an sql upgrade script available/needed for this version (currently 1.6.5.).

I would like to test the patch nonetheless, but I'm a bit lost where to actually apply the patch …

`/usr/share/webmin/virtual-server/scripts/` ?

Do I need to restart webmin?

**Comment by jcameron** [comment]
We've updated the Roundcube installer to version 1.6.6., so your system should pick up the new version of the installer within a day..
